Packs/Base/ReleaseNotes/1_11_10.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### SaneDocReports
+- Fixes incorrect bar chart values.
```

Comment:
Not according to our release notes templates. Please use `demisto-sdk doc-review ...`. Also please add a line stating that the docker image has been updated.